### ISSUE TITLE
Add an output port on the DashDB Out node

### ISSUE THREAD

**Issue by tonykambo** [smchamberlin/node-red-contrib-ibm-db2#7]
It would be ideal to have an output port on the DashDB Out node so that a flow could continue on and perform additional steps such as:
- Determine if the "insert" step from DashDB Out was successful or an SQL error was returned (for example, attempting to add a row with a duplicate value on a UNIQUE column). This would allow adding a switch to return either an HTTP 200 or an HTTP 409 for duplicate error.
- Add Dash DB In node straight after to retrieve the auto incremented index value for the row just added and then use that index to insert a row into a different table using that index as a foreign key. This would be part of a single HTTP In REST call that requires part of the payload to be inserted into one table and another part of the day inserted into another table.

This would be in the context of including DashDB Out as part of a REST service following an HTTP In node. Whilst the DashDB In node could be used to perform an SQL Insert it does not easily cater for passing in a JSON array for inserting multiple rows within the one payload. This is a handy feature of the DashDB Out node.

So in short, add an output port to the DashDB Out node to allow the flow to continue and pass on any SQL errors.

**Comment by tverilytt** [comment]
Hi!
This sounds like a very suiting request. Tried to add a Catch node, but it did not get SQL errors. An output port would definately make error handling easier, and to, when appropriate, return some information to caller / client.

Cheers
-jo

**Comment by tverilytt** [comment]
Created a PR for this https://github.com/smchamberlin/node-red-nodes-cf-sqldb-dashdb/pull/15

**Comment by tverilytt** [comment]
@smchamberlin - Any thoughts on this and the pull request for output node?

**Comment by smchamberlin** [comment]
@tverilytt If we were properly catching the errors for use in the Catch node (another PR open for that), would you still need extra output node?  I think stylistically it's weird to have an output node coming from an output node ... Thanks.

**Comment by tverilytt** [comment]
@smchamberlin That sounds like a feasible solution to me!
